### ISSUE TITLE
function-max-empty-lines

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#894]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Rule for disallow empty lines inside function parentheses.

> Which rule, if any, is this issue related to?

New rule `function-parentheses-disallow-empty-line-inside`

> What CSS is needed to reproduce this issue?

```css
.test {
    transform:
        translate(

            1,

            1

        );
}

.test-1 {
    transform:
        translate(
            1,

            1

        );
}

.test-2 {
    transform:
        translate(
            1,

            1
        );
}

.test-3 {
    transform:
        translate(

            1,
            1

        );
}

and etc
```

> Which version of stylelint are you using?

`4.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Throw error if empty line present inside function parentheses.

**Comment by davidtheclark** [comment]
Sure! PR welcome. I think it would have to be renamed to `function-parentheses-no-empty-line-inside`.

**Comment by jeddy3** [comment]
> Sure! PR welcome.

Agreed. Feel free to contribute this @evilebottnawi. It looks like a good rule to me. 

Shall we go with `function-max-empty-lines: int` for consistent with [`max-empty-lines`](http://stylelint.io/user-guide/rules/max-empty-lines/)? 

For example, if a user wants a maximum of one empty line between everything, but no empty lines with functions, they can do so with:

```json
{
  "max-empty-lines": 1,
  "function-max-empty-lines": 0
}
```

**Comment by davidtheclark** [comment]
Sure that's fine by me.

**Comment by ntwb** [comment]
Wouldn't [`function-parentheses-newline-inside`](https://github.com/stylelint/stylelint/tree/master/src/rules/function-parentheses-newline-inside) already cover this?


```json
{
  "rules": {
    "function-parentheses-newline-inside": "always-multi-line"
  }
}
```

**Comment by jeddy3** [comment]
> Wouldn't function-parentheses-newline-inside already cover this?

I believe they would work together e.g.:

```json
{
  "function-parentheses-newline-inside": "always-multi-line",
  "function-max-empty-lines": 0
}
```

Would allow:

```css
.test-2 {
    transform:
        translate(
            1,
            1
        );
}
```

But warning with:

```css
.test-2 {
    transform:
        translate(
            1,

            1
        );
}
```

**Comment by ntwb** [comment]
Cool, thanks :+1:
